java/src/com.zeroc.ice/src/main/java/com/zeroc/Ice/OperationInterruptedException.java

```diff
@@ -4,10 +4,6 @@
 
 /** This exception indicates a request was interrupted. */
 public final class OperationInterruptedException extends LocalException {
-    public OperationInterruptedException() {
```

Comment:
Removed this constructor to make sure we always provide the cause (= InterruptedException).